pkg/service/presentation/model/model.go

```diff
@@ -124,3 +121,50 @@ func ServiceModel(storedSubmission *storage.StoredSubmission) Submission {
 		VerifiablePresentation: &storedSubmission.VerifiablePresentation,
 	}
 }
+
+type CreateRequestRequest struct {
+	PresentationRequest Request `json:"presentationRequest"`
+}
+
+type CreateRequestResponse struct {
+	PresentationRequest Request `json:"presentationRequest"`
```

Comment:
do we need to return it back, or is a 201 ok?

Reply:
It's a more RESTful pattern. Instead of doing an additional GET, you send back what the GET will return.

Reply:
I'm ok with this but it's not in line with our other APIs